gimvi_tutorial.ipynb

```diff
@@ -9,7 +9,7 @@
    },
```

Comment:
<p>I think you have to specify <code>use_rep="X"</code>	in the call to neighbors</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>